streams/src/main/java/org/apache/kafka/streams/kstream/internals/CogroupedKStreamImpl.java

```diff
@@ -89,6 +92,20 @@
         return aggregate(initializer, Materialized.with(keySerde, null));
     }
 
+    @Override
+    public <W extends Window> TimeWindowedCogroupedKStream<K, VOut> windowedBy(final Windows<W> windows) {
+        Objects.requireNonNull(windows, "windows can't be null");
```

Comment:
Seems there is not test for this `null` check?

Reply:
fixed